vendor/engines/c2po/config/locales/en.yml

```diff
@@ -46,6 +46,7 @@ en:
     attributes:
       purchase_order_account:
         account_number: Payment Source
+        ar_number: Account Receivable Number
```

Comment:
I think we should edit the key on line [26](https://github.com/tablexi/nucore-open/pull/3266/files#diff-a6b0d91195ef79c1415d24e9d1d0fa1d6015b113b07d88c6072cb40b392415cbR26) instead.  What do you think?

Reply:
@jossim thoughts?